### PR TITLE
docs: really fix link

### PR DESCRIPTION
**Pull request by vinckr**
https://github.com/ory/meta/pull/63

### CODE REVIEW

File: templates/repository/common/CONTRIBUTING.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ Thank you for contributing changes to this document! Because we use a central re
 to synchronize this file across all our repositories, make sure to make your edits
 in the correct file, which you can find here:
 
-https://github.com/ory/meta/blob/master/templates/repository/CONTRIBUTING.md
+https://github.com/ory/meta/blob/master/CONTRIBUTING.md
```
Comment (aeneasr) [inline templates/repository/common/CONTRIBUTING.md:7]: This is not the correct file!

```suggestion
https://github.com/ory/meta/blob/master/templates/repository/common/CONTRIBUTING.md
```

File: templates/repository/common/CONTRIBUTING.md [hunk 2]
```diff
@@ -55,7 +55,7 @@ do this is via the [ORY Community](https://community.ory.sh/) or join the
 ## FAQ
 
 - I am new to the community. Where can I find the
-  [ORY Community Code of Conduct?](CODE_OF_CONDUCT.md)
+  [ORY Community Code of Conduct?](https://github.com/ory/meta/blob/master/CODE_OF_CONDUCT.md)
```
Comment (aeneasr) [inline templates/repository/common/CONTRIBUTING.md:58]: We should link to the project's code of conduct.

**Review by aeneasr — CHANGES_REQUESTED** [review]

File: templates/repository/common/CONTRIBUTING.md (hunk 2 above)
Comment (vinckr) [reply templates/repository/common/CONTRIBUTING.md:58]: Then we have to copy the code of conduct into each projects /docs too right?

Comment (aeneasr) [reply templates/repository/common/CONTRIBUTING.md:58]: https://github.com/ory/hydra/blob/master/CODE_OF_CONDUCT.md

**Comment by vinckr** [comment]
Added 
https://github.com/ory/{{Project}}/blob/master/CODE_OF_CONDUCT.md
for the coc

**Review by aeneasr — APPROVED** [review]

**Pull request merged**